docs/testing.md

```diff
@@ -114,3 +114,12 @@ go test -timeout=30m -v -tags=deployrancher ./entrypoint/deployrancher/... \
 ```
 chartsArgs rancherImage=<image or url>,extraEnv[0].name=CATTLE_AGENT_IMAGE,extraEnv[0].value=<image or url>-agent:v2.7.12
 ```
+
+### Testing with kubeconfig file.
+
+- Required variables in `.env` file
+```
+KUBE_CONFIG=<kubeconfig file 64 base encoded>
+BASTION_IP=<bastion public ip> when testing Dual-Stack
+AWS_USER_LOGIN=<aws user login> for ssh operations needed by `RunCommandOnNode()`
```

Comment:
Can we re-purpose `aws_user` variable from tfvars file?

Reply:
people can use their preference OS, meaning i dont need to run in the same Os, so we need to ask that info

Reply:
I mean we are passing the `aws_user` in the tfvars when running this, right? Can't we re-user that var in the test?

Reply:
totally true
fixed